third_party/k8s.io/kubernetes/hack/verify-licenses.sh

```diff
@@ -74,7 +74,12 @@ do
 	then
 		if [[ "${LICENSE_URL}" == 'Unknown' ]];
 		then
-			if  [[ "${GO_PACKAGE}" != k8s.io/* ]];
+			if [[ "${GO_PACKAGE}" == "sigs.k8s.io/controller-runtime" ]] && [[ "${LICENSE_NAME}" == "Apache-2.0" ]];
```

Comment:
is this from Alex's PR? does it need to be reverted?

Reply:
I could not find this in the repo. So I added it in order to fix the license-lint errors.

Reply:
let's revert this one for now, since we know it's due to fork and it's an optional CI check

Reply:
Ok. Thanks

Reply:
@sozercan Looks like it's not an optional CI check. I still get "Merging is Blocked".